components/MessageCard.vue

```diff
@@ -2,10 +2,12 @@
   <v-card class="messageCard">
     <v-list-item three-line>
       <v-list-item-content>
-        <div v-html="messageWithHTMLTag"/>
+        <div v-html="messageWithHTMLTag"/><br>
```

Comment:
brタグを使うより、スタイルでマージンを調整すると良いです
```suggestion
        <div v-html="messageWithHTMLTag"/>
```

Reply:
修正しました。https://github.com/takuuuu517/nuxt-sample-app/pull/5/commits/c8dc2273aa5dda8868ed1c078d2b469f0201123c

もともと改行したかっただけで、messageWithHTMLTagのしたがdivになったのでbrは必要なくなりました。brタグを消して修正しました。